xpk.py

```diff
@@ -263,11 +263,12 @@ class CapacityType(enum.Enum):
         labels:
           xpk.google.com/workload: {args.workload}
       spec:
-        backoffLimit: 0
+        backoffLimit: 4
```

Comment:
Can we have a special exit code for when pathways workers are dying after suspension? We should ideally not have that restart count towards the backofftLimit. I am ok if you wan to address that in a separate PR.

Reply:
This would likely be a much more involved change as backoffLimit is purposely generic to all failures. I think making a change specifically for suspension would still impact the backoffLimit, or rather show the job as completed (would not be intended).

I don't think it's a change we'd want to support in the long term either. I'm open to make this value tunable for the user though in the future.

Reply:
I thought GKE supported configuring the error types which should count towards your backoffLimit. My concern here would be that we would only be able to tolerate as many planned failures as the backoffLimit and that may be an issue for larger jobs.

Reply:
It looks like GKE does not consider SIGTERMs against the backoffLimit. I think we can explore this option as well in a different CL if we want to consider SIGTERM counting against restarts.